src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/StructUtils.vb

```diff
@@ -0,0 +1,264 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System
+Imports System.Globalization
+Imports System.Diagnostics
+Imports System.Reflection
+
+Imports Microsoft.VisualBasic.CompilerServices.ExceptionUtils
+Imports Microsoft.VisualBasic.CompilerServices.Utils
+
+Namespace Microsoft.VisualBasic.CompilerServices
+
+    Friend Interface IRecordEnum
+        Function Callback(ByVal FieldInfo As System.Reflection.FieldInfo, ByRef Value As Object) As Boolean
+    End Interface
+
+    ' TODO: Change to Friend
+    <System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)>
+    Public Class StructUtils
```

Comment:
Please do not add any new publicly visible API. Once API is public, it implicitly is documented and supported unless (rarely) noted otherwise, which means we cannot change it.

As you can see by your test failures, tests trying t ocall this will still fail when running against .NET Framework.

Please leave visibility as Friend and instead test everything through the public member `Strings.Len(object)`

Reply:
The API won't be extended, because commit *'DROP Make StructUtils.GetRecordLength public'* is temporary. I needed it to test `StructUtils`. 'DROP' has been used in the commit message because it's the rebase action to be taken.

Testing everything through `Strings.Len(object)` is under development...